tox.ini

```diff
@@ -93,15 +93,15 @@ commands=
 commands=pytest --doctest-modules _pytest
 deps=
 
-[testenv:doc]
+[testenv:docs]
 basepython=python
 changedir=doc/en
-deps=sphinx
-     PyYAML
+deps=
+    sphinx
+    PyYAML
 
 commands=
-    make clean
-    make html
+    sphinx-build -W -b html . _build
```

Comment:
Hmm just realized we still depend on it because of `regendoc`. 

Probably it would be better to get rid of it and move pertinent tasks to `tox`, but we can do that in a separate PR.